frontend2/src/contexts/EpisodeContext.ts

```diff
@@ -27,3 +28,8 @@ export const useEpisodeId = (): EpisodeIdContextType => {
   }
   return episodeIdContext;
 };
+
+/** Use this function to retrieve all episodes. */
```

Comment:
I'd add a comment here saying that it is undefined if the data hasn't loaded yet

Reply:
Interesting... I think that this is pretty standard across our frontend APIs? Like inferrable from "Maybe"? If you think it's useful though ig we can add :+1:

Reply:
That's true - up to you then!

Reply:
Think I'll omit it... if someone gets confused I'll add it haha